crates/re_ui/src/list_item2/property_content.rs

```diff
@@ -0,0 +1,278 @@
+use crate::list_item2::{ContentContext, ListItemContent};
+use crate::{Icon, ReUi};
+use eframe::emath::{Align, Align2};
+use eframe::epaint::text::TextWrapping;
+use egui::{NumExt, Response, Ui};
+
+/// Closure to draw an icon left of the label.
+type IconFn<'a> = dyn FnOnce(&ReUi, &mut egui::Ui, egui::Rect, egui::style::WidgetVisuals) + 'a;
+
+/// Closure to draw the right column of the property.
+type PropertyValueFn<'a> =
+    dyn FnOnce(&ReUi, &mut egui::Ui, egui::style::WidgetVisuals) -> Option<egui::Response> + 'a;
```

Comment:
PR3 will remove the return value as well, as its used was only to be forwarded to `ListItemContent::ui()`, which no longer needs it.